libraries/AP_DDS/dds_xrce_profile.xml

```diff
@@ -253,4 +253,19 @@
       <dataType>tf2_msgs::msg::dds_::TFMessage_</dataType>
     </topic>
   </data_reader>
+  <topic profile_name="velocitycontrol__t">
+    <name>rt/ap/velocity_control</name>
+    <dataType>geometry_msgs::msg::dds_::TwistStamped_</dataType>
+    <historyQos>
+      <kind>KEEP_LAST</kind>
+      <depth>5</depth>
+    </historyQos>
+  </topic>
+  <data_reader profile_name="velocitycontrol__dr">
+    <topic>
+      <kind>NO_KEY</kind>
+      <name>rt/ap/velocity_control</name>
```

Comment:
change to `/rt/ap/cmd_vel` to keep it similar to turtlebot3 (https://wiki.ros.org/turtlebot3_example#Published_Topics-1) and turtlesim (https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Launching-Multiple-Nodes/Launching-Multiple-Nodes.html)

Reply:
Done in e30ce00cf343e44ea6d79f3ee208467d4594bd0f